addons/mail/tools/discuss.py

```diff
@@ -37,29 +37,39 @@ def wrapper(self, *args, **kwargs):
 
     return wrapper
 
-def get_twilio_credentials(env) -> (str, str):
+
+def get_twilio_credentials(env) -> tuple[str, str] | None:
     """
     To be overridable if we need to obtain credentials from another source.
     :return: tuple(account_sid: str, auth_token: str)
     """
     params = env["ir.config_parameter"].sudo()
     account_sid = params.get_param("mail.twilio_account_sid")
     auth_token = params.get_param("mail.twilio_account_token")
+    if not params.get_param("mail.use_twilio_rtc_servers"):
+        return None
```

Comment:
I would return earlier and not even check the token/sid if the option is false, also return None, None so that it can still be unpacked, otherwise it will crash at:

```py
(account_sid, auth_token) = get_twilio_credentials(self.env)
```